ui/scripts/modal.js

```diff
@@ -23,6 +34,7 @@ export default class Modal {
         window.addEventListener('click', (e) => this.closeModalOutside(e));
     }
 
+    // @to-do: handle the case if trailer is not available
```

Comment:
don't you already check for `if (this.trailer_url) {` elsewhere before creating the Modal? or is this to-do about something else?

Reply:
Yes, I did it already in movie.js, just forgot to clean up the comment here. It seems more logical not even create modal component in case if the trailer is not available.